packages/clerk-js/src/ui/components/OrganizationList/utils.ts

```diff
@@ -0,0 +1,14 @@
+import type { useCoreOrganizationList } from '../../contexts';
+
+export const organizationListParams = {
```

Comment:
❓ What's the use of these hardcoded params here? 🤔

Reply:
So swr fetches data and cached them based on a key.

As key we use these params. For example 
`useOrganizatioList({userMemership:true})` will store data in a different place than `useOrganizationList({userMemership:{infinite:true}})`.

Inside OrganizationList we call useOrganizationList many times and for consistency I've added this obj as a source of truth for all of them.